Sources/RxAlamofire.swift

```diff
@@ -775,11 +775,11 @@ extension Reactive where Base: DataRequest {
                 case .success(let result):
                     if let httpResponse = packedResponse.response {
                         observer.on(.next(httpResponse, result))
+                        observer.on(.completed)
                     }
                     else {
                         observer.on(.error(RxAlamofireUnknownError))
                     }
-                    observer.on(.completed)
```

Comment:
I don't think it's orthogonal :) 
If it was supported (multiple next events), this will break it. 

But - to be honest I think it was never supported with this method and actually this would drive us closer to moving to Single so I'm all for supporting the most common use case. 

👍 from me